src/cli/cli_dataset.cpp

```diff
@@ -51,12 +51,12 @@ otError Dataset::Print(otOperationalDataset &aDataset)
 {
     if (aDataset.mComponents.mIsPendingTimestampPresent)
     {
-        OutputLine("Pending Timestamp: %lu", aDataset.mPendingTimestamp);
+        OutputLine("Pending Timestamp: %lu", aDataset.mPendingTimestamp.mSeconds);
```

Comment:
Ideally better to update the cli output to represent in secs, ticks, authoriative, maybe in separate PR given the timeline and possible impact on tests. For now,  may at least update
 https://github.com/openthread/openthread/blob/main/src/cli/README_DATASET.md to explicitly note that the output are only seconds part?

Reply:
I would prefer to keep the CLI as it is.